src/templates/layouts/staffProfile/index.test.tsx

```diff
@@ -123,4 +124,67 @@ describe('StaffProfile Component', () => {
 
     expect(screen.queryByText(/Download full bio/)).not.toBeInTheDocument()
   })
+
+  describe('Lovell variants - tricare', () => {
+    const modifiedProfile = {
+      ...mockProfile,
+      menu: {
+        depth: 5,
+        link: { label: 'Prachi Asher', url: { path: '/' }, links: [] },
+        parent: {
+          label: 'Leadership',
+          links: [],
+          url: { path: '/leadership' },
+        },
+      },
+      vamcOfficalName: 'Lovell Federal health care',
+      lovellVariant: 'tricare' as LovellChildVariant,
+      lovellSwitchPath: '/lovell-federal-health-care-va/leadership',
+    }
+    test('LovellSwitcher is rendered', () => {
+      render(<StaffProfile {...modifiedProfile} />)
+
+      expect(
+        screen.getByText('You are viewing this page as a TRICARE beneficiary.')
+      ).toBeInTheDocument()
+    })
+    test('shows the correct VAMC official name', () => {
+      render(<StaffProfile {...modifiedProfile} />)
+
+      expect(
+        screen.getByText('Lovell Federal health care - TRICARE')
+      ).toBeInTheDocument()
+    })
+  })
+  describe('Lovell variants - va', () => {
+    const modifiedProfile = {
+      ...mockProfile,
+      menu: {
+        depth: 5,
+        link: { label: 'Prachi Asher', url: { path: '/' }, links: [] },
+        parent: {
+          label: 'Leadership',
+          links: [],
+          url: { path: '/leadership' },
+        },
+      },
+      vamcOfficalName: 'Lovell Federal health care',
+      lovellVariant: 'va' as LovellChildVariant,
```

Comment:
Alternatively, you can slap a type declaration on `modifiedProfile` or an `as const` at the end of the object literal and TS will be less annoyed that this could be any ol' string.
```ts
const modifiedProfile: FormattedStaffProfile = { // Either here
  ...
} as const // Or here
```

I try to avoid type assertions wherever possible because they smell. 🙊 🙈 In this case, it's fine for now, but if for whatever reason we remove `'va'` from that `LovellChildVariant` union later, this won't throw a TS error like it should.

Reply:
Still getting the hang of TS. Thanks this is super helpful. Sometimes i'm just confused about what it wants. 🫠